packages/theme/src/Theme/dashboard/component-changes.tsx

```diff
@@ -32,6 +32,9 @@ export const Grid: Theme = {
 
 const button = css_var('m-button', { main: 1, contrast: 1, light: 1 })
 export const Button: Theme = (mode, colors): ThemeOptions => ({
+    typography: {
+        fontFamily: 'Helvetica',
```

Comment:
Please remove all component level font family rules as 'fontFamily: 'Helvetica'.

Reply:
OK. Removed that in popup

Reply:
there is no font fallback? how will Windows/Linux users see this